framework/ipc/src/org/apache/cloudstack/framework/serializer/OnwireClassRegistry.java

```diff
@@ -108,7 +107,7 @@ public void scan() {
                 while (resources.hasMoreElements()) {
                     String filePath = resources.nextElement().getFile();
                     // WINDOWS HACK
-                    if (filePath.indexOf("%20") > 0)
+                    if (filePath != null && filePath.indexOf("%20") > 0)
```

Comment:
I didn't do that as there is a new assignment to the var in the if block. The replaceAll contract doesn't state a null retrun as possibiliy so I'll move the replaceAll inside the next if block if the rest of the flow allows it